cfme/containers/provider/openshift.py

```diff
@@ -328,6 +328,20 @@ def sync_ssl_certificate(self):
              None
         """
 
+        def _copy_certificate():
+            is_succeed = True
+            try:
+                # Copy certificate to the appliance
+                provider_ssh.get_file("/etc/origin/master/ca.crt", "/tmp/ca.crt")
+                appliance_ssh.put_file("/tmp/ca.crt",
+                                       "/etc/pki/ca-trust/source/anchors/{crt}".format(
+                                           crt=cert_name))
+            except Exception:
```

Comment:
Required: Please specify more exact type of Exception

Reply:
@rlbabyuk  Well you right, the exception that I need to catch here is ```URLError```,

I found that this exception is in different packages in Python 2.7 and 3.6.

In Python 3.6 you can find this exception in ```urllib.error``` and in Python 2.7 it exists in ```urllib2```

Do you know how can I manage this situation?

Reply:
@rlbabyuk Do you think that is a good solution?

```python
try:
    from urllib2 import URLError
except ImportError as ex:
    from urllib.error import URLError
```

Reply:
please use import from six.moves.urllib.error import URLError

Reply:
@rlbabyuk Done